server/datastore/mysql/apple_mdm.go

```diff
@@ -750,6 +750,11 @@ func updateMDMAppleHostDB(
 	mdmHost *fleet.Host,
 	appCfg *fleet.AppConfig,
 ) error {
+	refetchRequested := 1
+	if mdmHost.Platform == "ios" || mdmHost.Platform == "ipados" {
```

Comment:
Might be worth refactoring in a `Host.SupportsRefetch()` method to centralize this condition?

Reply:
Will add a `host.SupportsOsquery`.